psalm-baseline.xml

```diff
@@ -1,14 +1,138 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <files psalm-version="4.7.3@38c452ae584467e939d55377aaf83b5a26f19dd1">
+  <file src="src/Annotation/AbstractBuilder.php">
+    <InvalidArrayOffset occurrences="1">
+      <code>$params['elementSpec']['spec']['type']</code>
+    </InvalidArrayOffset>
+  </file>
+  <file src="src/Annotation/AttributeBuilder.php">
+    <NullArgument occurrences="2">
+      <code>$annotations</code>
+      <code>$annotations</code>
+    </NullArgument>
+    <UndefinedMethod occurrences="2">
+      <code>getAttributes</code>
+      <code>getAttributes</code>
+    </UndefinedMethod>
+  </file>
   <file src="src/ConfigProvider.php">
     <DuplicateArrayKey occurrences="2">
       <code>\Zend\Form\Annotation\AnnotationBuilder::class =&gt; Annotation\AnnotationBuilder::class</code>
       <code>\Zend\Form\FormElementManager::class           =&gt; FormElementManager::class</code>
     </DuplicateArrayKey>
   </file>
+  <file src="src/Element.php">
+    <InvalidPropertyAssignmentValue occurrences="1">
+      <code>$messages</code>
```

Comment:
I'm going to fix most of this array-related errors in a next PR by moving `array|Traversable` to `iterable`

Reply:
See https://github.com/laminas/laminas-form/pull/125/files#diff-3208225af5a3ef0c924761633ad6b04cbfa5eab3d3bafcc49c79441e89257ee7